### DISCUSSION TITLE
Retroscaler's PS1 adapter

### DISCUSSION THREAD

**Discussion by CharlieNash00** [darthcloud/BlueRetro#664] [Q&A] [answered]
hi I tried using Retroscaler's ps1 adapter and it didn't work on three PS3 controllers I own. I couldn't get it to connect to blueretro.io website.

What I did after to update the firmware and retrieve the sixaxis code was the other solution provided using this software they include with the firmware download called (nodeMCUpyflasher) that worked... and I was able to get the sixaxis code to set to the controller but the sixaxis ps3 controller still didnt work... after trying both firmwares they offered.

https://www.retroscaler.com/archives/415

Im not sure if the guy on aliexpress selling these adapters is officially associated with retroscaler? or what??? but the support has been dismal to help me get this thing working. This should be very straight forward but hasnt and ive followed the instructions very carefully and spent too much time on this then Ive already wanted.

I've tried so far one **SIXAXIS** ps3 controller and two **DUALSHOCK 3** ps3 controllers with no success with this adapter. It works with more modern controllers ive tried but not the ps3 controllers. 

does the blue retro project just support **sixaxis ps3** controllers or does it support **dualshock 3** also??? i cant find any information on this. 

how would I pair a dual shock 3 controller since the sixaxis software doesnt detect those controllers? I just want to use my ps3 controller on my ps2... 

Not sure also why blueretro.io website wont detect the retroscaler ps1/ps2 adapter? I have it connected to my laptop and the computer detects the adapter and shows it in device manager but the website absolutely doesnt detect it... any solutions would be appreciated.

**Comment by darthcloud** [comment]
The wiki list literally list both Sixaxis and DS3 in the list
https://github.com/darthcloud/BlueRetro/wiki/Controller-pairing-guide#1---list-of-tested-bluetooth-devices

The sixaxis application work fine with DS3.

If it doesn't work you likely got counterfeit chinese controllers

**Reply by CharlieNash00** [reply]
My sixaxis without a doubt is authentic. It came with my launch playstation 3. The other two controllers have the sony logo on them with the serial stickers but i havent opened them up to verify since buying them used. It's definitely got to be the issue with your website or bluescalers adapter.

Are you able to just release a software based blueretro.io? because I cant get 3 controllers to connect to it.  At least one of the controllers should work.

**Reply by darthcloud** [reply]
controller don't connect to the website. You go on website only to get the BlueRetro mac address.

You then connect your controller to a pc running the sixaxis tool and program that address in the controller. after you disconnect the usb cable and connect to BlueRetro with the ps buttons.

**Reply by CharlieNash00** [reply]
has to be the retroscaler adapter because it doesnt connect to the blueretro.io website
I updated the sixaxis code with their nodeMCU program since it wouldnt connect to the website but still the adapter wont pair to the sixaxis controller.

**Comment by pceslayer** [comment]
You cannot pair a PS3 controller the same way you pair a normal Bluetooth
controller.

Are you using the PC tool to set the MAC address on your PS3 controllers
first before trying to connect to blue retro?

**Reply by CharlieNash00** [reply]
yes I used the nodeMCU program to obtain the MAC address on the controllers, changed them to match the retroscaler blueretro adapter I bought off aliexpress using the SIXAXIS software and no luck. 

I tried using the BlueRetro.IO website to obtain the MAC addresses but the website doesnt work. These controllers are official playstation controllers, not fake.

**Comment by CharlieNash00** [comment, marked as answer]
I finally found a solution. After spending days on this annoyance here's what I did...

Don't follow the RetroScaler manual guide. **Avoid using the NodeMCU tool to try to update the firmware recommended in the retro scaler manual!**

The MAC address given to me from the adapter was wrong after updating the firmware using the nodeMCU program because I tried it as an alternative because BlueRetro.IO wasnt working on my pc or laptop... don't know why??? that's an issue with the website that needs to be sorted out. My bluetooth works perfectly for other devices being detected on my laptop because it has the latest OS where as my pc doesnt. I also tried different web browsers and didn't work either. Anyways...

STEP 1:
So what I did was I tried to connect to the retroscaler ps2 adapter powered via usb jack through my android phone on blueretro.io. 

STEP 2: I downloaded the HW1 latest firmware onto my android phone. unzipped the files and uploaded it on the firmware option on blueretro.io and then updated it on there.

STEP 3: after the device updated to the latest firmware I clicked "BlueRero System Manager" on blueretro.io via my android phone browser and it gave me the MAC address. The MAC address strangely had two digits off from what the firmware that the NodeMCU program gave me after updating the firmware using that and when I checked it via the SIXAXIS program. 

STEP 4: updated the mac address with the slightly different mac address given with the firmware updated using blueretro.io this time.

STEP 5: plugged in the ps2 retroscaler adapter to my ps2 and my ps3 controller instantly connected to it. _**WINNING!**_ 



All and all my suggestions is create an offline program version of blueretro.io as another option to properly update the firmware since the web based version on pc and laptops doesnt work right at least for me and I got lucky the the android based version of it on my phone worked as another alternative.

Suggest that the aliexpress sellers like retroscaler dont put in their manual to use NODEmcu to update the firmware because that clearly messed up the device providing a wrong mac address.
